app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Open Event Organizer</string>
+    <string name="app_name">eventyay organizer</string>
```

Comment:
@niranjan94 Please suggest a solution. We'll have to add app name in either build.gradle or strings.xml (refrerred from AndroidManifest.xml)

Configuration may be done via environment variable and switched to default string if it is not present. Is this OK?